backend/internal/database/database.go

```
@@ -284,9 +279,74 @@ CREATE INDEX IF NOT EXISTS provider_imports_instance_idx ON provider_imports(ins
+CREATE INDEX IF NOT EXISTS api_keys_token_hash_idx ON api_keys(token_hash);
+`
+	_, err := db.ExecContext(ctx, schema)
+	return err
+}
+
+func tableHasColumn(ctx context.Context, db *sql.DB, table, column string) (bool, error) {
+	rows, err := db.QueryContext(ctx, `PRAGMA table_info(`+table+`)`)
+	if err != nil {
+		return false, err
+	}
+	defer rows.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)